contracts/extensions/contracts/src/MaximumGasPrice.sol

```diff
@@ -0,0 +1,52 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.16;
+pragma experimental ABIEncoderV2;
```

Comment:
I don't think we actually need to use ABIv2 here. I'd just run tests with/without to see if there are any gas differences.

Reply:
looks like without is _slightly_ cheaper for the parametrized version